devices/tuya/_TZE200_dwcarsat_air_sensor.json

```diff
@@ -0,0 +1,212 @@
+{
+  "schema": "devcap1.schema.json",
+  "manufacturername": "_TZE200_dwcarsat",
+  "modelid": "TS0601",
+  "vendor": "Tuya",
+  "product": "Smart Air House keeper",
+  "sleeper": true,
+  "status": "Gold",
+  "subdevices": [
+    {
+      "type": "$TYPE_HUMIDITY_SENSOR",
+      "restapi": "/sensors",
+      "uuid": [
+        "$address.ext",
+        "0x01",
+        "0x0511"
+      ],
+      "items": [
+        {
+          "name": "attr/id"
+        },
+        {
+          "name": "attr/lastannounced"
+        },
+        {
+          "name": "attr/lastseen"
+        },
+        {
+          "name": "attr/manufacturername"
+        },
+        {
+          "name": "attr/modelid"
+        },
+        {
+          "name": "attr/name"
+        },
+        {
+          "name": "attr/type"
+        },
+        {
+          "name": "attr/uniqueid"
+        },
+        {
+          "name": "config/on"
+        },
+        {
+          "name": "config/reachable"
+        },
+        {
+          "name": "state/humidity",
+          "read": {
+            "fn": "none"
+          },
+          "parse": {
+            "dpid": 19,
+            "eval": "Item.val = (10 * Attr.val);",
+            "fn": "tuya"
+          },
+          "default": 0
+        },
+        {
+          "name": "state/lastupdated"
+        }
+      ]
+    },
+    {
+      "type": "$TYPE_AIR_QUALITY_SENSOR",
+      "restapi": "/sensors",
+      "uuid": [
+        "$address.ext",
+        "0x01",
+        "0x0512"
+      ],
+      "items": [
+        {
+          "name": "attr/lastannounced"
+        },
+        {
+          "name": "attr/lastseen"
+        },
+        {
+          "name": "attr/manufacturername"
+        },
+        {
+          "name": "attr/modelid"
+        },
+        {
+          "name": "attr/name"
+        },
+        {
+          "name": "attr/type"
+        },
+        {
+          "name": "attr/uniqueid"
+        },
+        {
+          "name": "config/on"
+        },
+        {
+          "name": "config/reachable"
+        },
+        {
+          "name": "state/airqualityco2",
+          "read": {
+            "fn": "none"
+          },
+          "parse": {
+            "dpid": 22,
+            "eval": "Item.val = Attr.val;",
+            "fn": "tuya"
+          },
+          "default": 0
+        },
+        {
+          "name": "state/airqualityformaldehyd",
```

Comment:
This is a typo, should be formaldehyd**_e_**

Reported here https://github.com/dresden-elektronik/deconz-rest-plugin/issues/6810

Reply:
I suggest to wait until #6673 is merged as the unit-type will change to `state/airquality_formaldehyde_density` (hopefully soon)

Reply:
Yep, it was corected here https://github.com/dresden-elektronik/deconz-rest-plugin/pull/6673
But the PR is not yet valided, so I m waiting before.